variables.tf

```diff
@@ -208,3 +208,9 @@ variable "skip_gcloud_download" {
   type        = bool
   default     = false
 }
+
```

Comment:
Wouldnt we need to expose `vpc_service_control_attach_enabled` as well and pass it in to core?